lib/rate-limiter.js

```diff
@@ -12,15 +12,19 @@
  * @return {type}     returns a function that takes config options object
  */
 module.exports = function(db, app) {
+
   return function(opts) {
     var middleware = function(req, res, next) {
+      if (!db) {
```

Comment:
Okay, just tested it out. And `!db || !db.connected` will pass through the request.